components/tabs/style/index.less

```diff
@@ -80,6 +80,7 @@
     }
 
     .@{tab-prefix-cls}-nav-add {
+      min-height: @tabs-card-height;
```

Comment:
Great work! But actually ng-zorro's styles are synchronized from ant-design, in this case, we can change the styles in patch.less instead of index.less until ant-design has resolved it.(In general we always follow ant-design interactions)

Reply:
nit: can you add some tests for this?

Reply:
The test has been added.The style define have moved to patch.less.Please review it.